app/enterprise/0.34-x/developer-portal/how-to/serve-assets/aws-cloudfront.md

```diff
@@ -52,3 +55,15 @@ Once your distribution is ready, find the image you want to use in your S3 Bucke
 
 You can now copy the `Object URL` for the image and use it in the Dev Portal.
 
+### FAQs
+
+1. Why am I getting a “Access Denied” when requesting the “logo.png” through my Cloudfront URL?
+
+    This is a DNS redirect that S3 does by default on newly created buckets. The bucket's DNS has not fully propagated to <name>.s3.amazonaws.com, you can do one of two things:
+
+    a. Wait for the DNS to propagate
+
+    b. Change the “Origin” on your “Cloudfront Distribution” to be the exact bucket URI such as <name>.s3-us-west-1.amazonaws.com
+
+    If you are still seeing this error after 48 hours, check that the image
+    has been set to `public` as seen in Step 9
```

Comment:
Since the FAQ is formatted with the Step styles, it is hard to visually see where the question ends and the answer begins.

We should change this to be a Q&A format.